test-cypress/steps/cluster/custer-nodes-configuration-steps.js

```diff
@@ -57,7 +57,7 @@ export class ClusterNodesConfigurationSteps {
     }
 
     static enterNodeEndpoint(endpoint) {
-        cy.get('.edit-node-row .location-cell textarea[name="location"]')
+        cy.get('.edit-node-row .location-cell input[name="location"]')
```

Comment:
This is why binding css selectors to DOM tree is a bad thing. You can remove the tag name and leave only the attribute selector.